postfix/init.sls

```diff
@@ -1,5 +1,16 @@
 {% from "postfix/map.jinja" import postfix with context %}
 
+{%- if grains['os_family']=="Suse" %}
```

Comment:
I see this is based on a line further below but I would suggest using the usual spacing (for that line as well):

```suggestion
{%- if grains['os_family'] == "Suse" %}
```

We generally use `grains.os_family` but keeping it as `grains['os_family']` is fine as well.

Reply:
I used the same syntax that the file had below. But it makes sense to change both then.